Sources/SwiftDocC/Utility/FoundationExtensions/String+Capitalization.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension StringProtocol {
+    
+    /// Returns the string with the first letter capitalized.
+    /// This auto-capitalization only occurs if the first word is all lowercase and contains only characters A-Z.
+    /// The first word can also contain punctuation (e.g. a period, comma, hyphen, semi-colon, colon).
+    var capitalizeFirstWord: String {
```

Comment:
minor: Since this is doing non-constant time work (allocations and iterating over the entire string) it's a common convention to use a function without arguments instead of a computed property to indicate to the caller that this may be an experience call.